src/IFrameNavigator.ts

```diff
@@ -177,13 +234,13 @@ export default class IFrameNavigator implements Navigator {
 
         let statusMessage = "";
         if (status === CacheStatus.Uncached) {
-            statusMessage = "Not available offline";
+            statusMessage = "Download";
```

Comment:
I would like to test removing all the labels and see if users have trouble with the controls...  what do you think about that?